### ISSUE TITLE
Configuration log level is ignored by GenerationTool for first log statements

### ISSUE THREAD

**Issue by etiennestuder** [jOOQ/jOOQ#10463]
### Expected behavior and actual behavior:

Setting `logging = org.jooq.meta.jaxb.Logging.WARN` in the xml config file for the jOOQ generation tool still prints the first log statement which is of level INFO:

```
Aug 01, 2020 4:46:04 PM org.jooq.tools.JooqLogger info
INFO: Initialising properties  : /some/folder/build/tmp/generateJooq/config.xml
Aug 01, 2020 4:46:05 PM org.jooq.tools.JooqLogger warn
WARNING: DEPRECATED               : The <expression/> element in <forcedType/> is deprecated. Use...
```
It seems to me that the generation tool always starts on INFO level and only changes to the configured logging level in a subsequent step. Which might be logical given the config file needs to be read before the log level can be determined.

### Versions:

- jOOQ: 3.13.4
- Java: 1.8
- Database (include vendor): H2
- OS: Mac
- JDBC Driver (include name if inofficial driver): com.h2database:h2:1.4.200

@lukaseder Lukas feel free to close this issue if this is a chicken-egg problem that you do not intend to address.

**Comment by lukaseder** [comment]
Thank you very much for your report.

There are some log messages in `GenerationTool::main`, which cannot read the configuration's log level. Most of them are fatal errors, which log on error level. This one, however, could be rewritten in a way to depend on the configuration:

```java
                log.info("Initialising properties", file);
                generate(load(in));
```

Will be an easy fix

**Comment by etiennestuder** [comment]
Thanks for considering it, @lukaseder .

**Comment by lukaseder** [comment]
There's another log message that is logged at `INFO` level before the logging threshold is set. When `-Djooq.codegen.skip=true`, there's a `Skipping jOOQ code generation` message

**Comment by lukaseder** [comment]
Fixed for jOOQ 3.14.0. Backport scheduled for 3.13.5 (#10468)

**Comment by etiennestuder** [comment]
Cool. Thanks for the quick fix.
